File: Programmers/lv1/lv1_정수 내림차순으로 배치하기.java
https://programmers.co.kr/learn/courses/30/lessons/12933

import java.util.*;

class Solution {
    public long solution(long n) {
        long answer = 0;
        
        String str = Long.toString(n);
        String[] arr = str.split("");
        
        Arrays.sort(arr, Collections.reverseOrder());
        String tmp = String.join("",arr);   //띄어쓰기 없이 배열을 문자열로 변환
        answer = Long.parseLong(tmp);
        //System.out.println((answer));
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_정수 내림차순으로 배치하기.cs
using System;

public class Solution
{
    public long solution(long n)
    {
        long answer = 0;

        String str = n.ToString();
        Char[] temp = str.ToCharArray();

        Array.Sort(temp);
        Array.Reverse(temp);

        answer = long.Parse(temp);

        return answer;
    }
}